simulation_server.py

```diff
@@ -637,7 +637,9 @@ def percent(value):
         swap = psutil.swap_memory()
         if nvidia:
             nvmlHandle = nvmlDeviceGetHandleByIndex(0)
-            gpu = nvmlDeviceGetName(nvmlHandle).decode('utf-8')
+            gpu = nvmlDeviceGetName(nvmlHandle)
+            if isinstance(gpu, bytes):
+                gpu = gpu.decode('utf-8')
```

Comment:
This problem was probably caused by a very recent change (5 days ago) in the pynvml module:
https://github.com/gpuopenanalytics/pynvml/pull/46/files#diff-c9c6160b3107b52ed919117b3cba3bd81f8539baa056cf1e8693329a26ba5bc7R2009

Reply:
Yes, so I think it is better to keep the `isinstance(gpu, bytes)` check even so now I think the return type is string as it could change again.